WooCommerce/Classes/ViewRelated/Products/Variations/Edit Attributes/RenameAttributesViewModel.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+import Yosemite
+
+final class RenameAttributesViewModel {
+
+    let attributeName: String
+
+    private(set) var newAttributeName: String?
+
+    init(attributeName: String) {
+        self.attributeName = attributeName
+    }
+
+}
+
+// MARK: - Actions
+extension RenameAttributesViewModel {
+
+    func handleAttributeNameChange(_ name: String?) {
+        guard name != nil && name?.isNotEmpty == true else {
+            newAttributeName = nil
+            return
+        }
```

Comment:
I think you can drop this guard and just assign the content, or am I missing something? 😅

Reply:
Oh, yep! I was struggling to check for an empty text field to enable the Done button, so I was using this to check for `nil`. But I realized I can just assign the string here and then check for an empty string to enable the Done button:

```swift
self?.enableDoneButton(self?.viewModel.newAttributeName != "")
```